gcloud-java-dns/src/main/java/com/google/gcloud/dns/ChangeRequest.java

```diff
@@ -189,7 +189,7 @@ static ChangeRequest fromPb(Dns dns, String zoneName, Change pb) {
   static Function<Change, ChangeRequest> fromPbFunction(final Dns dns, final String zoneName) {
     return new Function<Change, ChangeRequest>() {
       @Override
-      public ChangeRequest apply(com.google.api.services.dns.model.Change pb) {
```

Comment:
Codacy in fact did not ask me to import it/not to import it. The message was that I was using a fully qualified name even though an import was present already.